driver/compile_common.mli

```diff
@@ -67,6 +67,15 @@ val typecheck_impl :
     its public interface.
 *)
 
+val implementation :
+  tool_name:string ->
+  native:bool ->
+  backend:(info -> Typedtree.structure * Typedtree.module_coercion -> unit) ->
+  sourcefile:string ->
+  outputprefix:string ->
+  unit
+(** The complete compilation pipeline for interfaces. *)
```

Comment:
Very minor copy paste error here (on my part): can you replace "interfaces" by "implementations"?

Reply:
Sure, done. Thanks for the catch.